arch/arm64/Kconfig

```
@@ -49,6 +49,14 @@ config ARCH_CHIP_FVP_ARMV8R
 	---help---
 		ARM FVP virt platform (ARMv8r)
 
+config ARCH_CHIP_IMX8
+	bool "NXP iMX.8 Platform (ARMv8a)"
```

```suggestion
	bool "NXP i.MX8 Platform (ARMv8a)"
```
See: https://www.nxp.com/products/processors-and-microcontrollers/arm-processors/i-mx-applications-processors/i-mx-8-applications-processors:IMX8-SERIES